<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="pushButton_commonPlot">
                <property name="text">
                 <string>Common</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_logPlot">
                <property name="text">
                 <string>Log</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
              <item>
               <widget class="QStackedWidget" name="stackedWidget_table">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="page_commonTable">
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="0" column="0">
                   <widget class="QTableView" name="tableView_commonTable"/>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="page_2"/>
               </widget>
              </item>
              <item>
               <widget class="QScrollArea" name="scrollArea">
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>69</width>
                   <height>273</height>
                  </rect>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_100">
                  <item row="0" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_showCurve">
                    <item>
                     <spacer name="verticalSpacer_7">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>340</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QStackedWidget" name="stackedWidget_plot">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="page_commonPlot">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QWidget" name="widget_commonPlot" native="true"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_logPlot">
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QWidget" name="widget_logPlot" native="true"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_350">
            <property name="spacing">
             <number>15</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox_59">
              <property name="title">
               <string>工作日志</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_63">
               <item row="0" column="0">
                <widget class="QTextBrowser" name="textBrowser_workLog"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_175">
              <property name="spacing">
               <number>15</number>
              </property>
              <item>
               <widget class="QGroupBox" name="groupBox_60">
                <property name="title">
                 <string>操作日志</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_65">
                 <item row="0" column="1">
                  <widget class="QTextBrowser" name="textBrowser_operatorLog"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line_13">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_61">
                <property name="title">
                 <string>异常日志</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_66">
                 <item row="0" column="0">
                  <widget class="QTextBrowser" name="textBrowser_errorLog"/>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>页</string>
         </attribute>
         <widget class="QWidget" name="widget_slide" native="true">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>140</y>
            <width>571</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton{
	background:transparent;
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_1">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>btn1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>btn2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>btn3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>btn4</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_dbSta">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1123</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
